kaiax/gov/param.go

```diff
@@ -371,6 +400,24 @@ var Params = map[ParamName]*Param{
 	},
 }
 
+var (
+	GovernanceAddValidator    ParamName = "governance.addvalidator"
```

Comment:
Could you update this file with the dev's? This logic has been added by #132

Reply:
Thanks, rebased it.

Reply:
Btw, I removed the `gov.ValidatorParamName` type because vote.Name() returns mismatched type error.. Do you have better idea?